docs/V4-Migration-Guide.md

```diff
@@ -25,6 +25,16 @@ If you are upgrading from a version before 3.0.0, please see the [previous migra
 
 ### Breaking Changes:
 
+#### [Goodbye Projector Mixin]()
+
+As part of the new vdom, we've removed the `ProjectorMixin`. The `ProjectorMixin` has been one of the more confusing aspects within dojo, especially due to the need to create an instance. This is fundamentally different to how widgets were used everywhere else in the framework.
```

Comment:
new vdom ... either use    new `<package name>`   or    new virtual dom implementation (with a link to the ticket)